bom/application/pom.xml

```diff
@@ -44,7 +44,7 @@
         <smallrye-health.version>3.2.1</smallrye-health.version>
         <smallrye-metrics.version>3.0.5</smallrye-metrics.version>
         <smallrye-open-api.version>2.1.22</smallrye-open-api.version>
-        <smallrye-graphql.version>1.6.0</smallrye-graphql.version>
+        <smallrye-graphql.version>1.6.2-SNAPSHOT</smallrye-graphql.version>
```

Comment:
This is my testing locally and will be changed before this PR is marked as ready for review.